app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsRobot.kt

```diff
@@ -73,6 +75,8 @@ class SettingsRobot {
     // PRIVACY SECTION
     fun verifyPrivacyHeading() = assertPrivacyHeading()
 
+    fun verifyHTTPSOnlyModeButton() = assertHTTPSOnlyModeButton()
+    fun verifyHTTPSOnlyModeValue(state: String) = assertHTTPSOnlyModeValue(state)
```

Comment:
I think it's clearer if we rename this function to `verifyHTTPSOnlyModeState`

Reply:
Done!